The linter failed:

```
main module (github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver) does not contain package github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver/examples/federation/prom-counter
Check License finished successfully
running misspell -error
golangci-lint run --allow-parallel-runners
Error: main.go:65:19: Error return value of `logger.Sync` is not checked (errcheck)
	defer logger.Sync()
	                 ^
make[2]: Leaving directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver/examples/federation/prom-counter'
```

--- a/receiver/simpleprometheusreceiver/examples/federation/prom-counter/main.go
+++ b/receiver/simpleprometheusreceiver/examples/federation/prom-counter/main.go
@@ -62,7 +62,9 @@ func main() {
 	meter := initMeter()
 	// logging
 	logger, _ := zap.NewProduction()
-	defer logger.Sync()
+	defer func() {
+		_ = logger.Sync()
+	}()
 	logger.Info("Start Prometheus metrics app")
 	valueRecorder, err := meter.SyncInt64().Histogram("prom_counter")
 	if err != nil {

